test-suites/gutenberg/sanity-tests.md

```diff
@@ -131,3 +131,31 @@ Cover-2
 - [ ] Cover - Replace media - [steps](https://github.com/wordpress-mobile/test-cases/blob/master/test-cases/gutenberg/cover.md#tc006)
 - [ ] Cover - Gradient background is rendered properly - [steps](https://github.com/wordpress-mobile/test-cases/blob/master/test-cases/gutenberg/cover.md#tc007)
 - [ ] Cover - Gradient overlay is rendered properly - [steps](https://github.com/wordpress-mobile/test-cases/blob/master/test-cases/gutenberg/cover.md#tc008)
+
+Columns - 1
+<!--- expected behaviour --->
+- [ ] Columns - Columns block is added with default number of Column - [steps](https://github.com/wordpress-mobile/test-cases/blob/master/test-cases/gutenberg/columns.md#tc001)
+- [ ] Columns - Adding Column with AppenderButton- [steps](https://github.com/wordpress-mobile/test-cases/blob/master/test-cases/gutenberg/columns.md#tc003)
+- [ ] Columns - Adding Column with Inserter (currently disabled) - [steps](https://github.com/wordpress-mobile/test-cases/blob/master/test-cases/gutenberg/columns.md#tc004)
```

Comment:
Same here 
```suggestion
- [ ] Columns - Adding Column with Inserter - [steps](https://github.com/wordpress-mobile/test-cases/blob/master/test-cases/gutenberg/columns.md#tc004)
```

Reply:
I have deleted it according to above comment